### ISSUE TITLE
Unicode in process.title causes failure

### ISSUE THREAD

**Issue by xiaoping0x** [macbre/analyze-css#136]
errors:

```
The header content contains invalid characters
```

cause: https://github.com/macbre/analyze-css/blob/devel/lib/runner.js#L25

```
function getUserAgent() {
	var format = require('util').format,
		version = require('../package').version;

	return format(
		'analyze-css/%s (%s %s, %s %s)',
		version,
		// Unicode process title cause faliure
		process.title,
		process.version,
		process.platform,
		process.arch
	);
}
```

**Comment by macbre** [comment]
Interesting, `process.title` should be `"node"`. How does `process` object look like in your case?

**Comment by xiaoping0x** [comment]
Run under windows administrator mode:

```
analyze-css --url http://localhost:3000
```

Error details:

```
  analyze-css:http Options: {"url":"http://localhost:3000","headers":{"User-Agent":"analyze-css/0.12.3 (管理员: C:\\Wind
ows\\system32\\cmd.exe - node_modules\\.bin\\analyze-css  --url http://localhost:3000 v6.9.1, win32 x64)"},"agent":{"dom
ain":null,"_events":{},"_eventsCount":1,"defaultPort":80,"protocol":"http:","options":{"path":null},"requests":{},"socke
ts":{},"freeSockets":{},"keepAliveMsecs":1000,"keepAlive":false,"maxSockets":null,"maxFreeSockets":256}} +1ms
```

**Comment by macbre** [comment]
Fixed by #137 and published to npm as `analyze-css@v0.12.4`
